.golangci.yml

```diff
@@ -88,7 +88,7 @@ linters-settings:
     disable:
       - shadow
     disable-all: false
-  golint:
+  revive:
     # minimal confidence for issues, default is 0.8
     min-confidence: 0.8
```

Comment:
Should `min-confidence` change to `confidence`? 
https://github.com/mgechev/revive#configuration
```
# Sets the default failure confidence. This means that linting errors
# with less than 0.8 confidence will be ignored.
```